cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -153,6 +150,10 @@ func (g *genDeepCopy) copyableWithinPackage(t *types.Type) bool {
 	if !strings.HasPrefix(t.Name.Package, "k8s.io/kubernetes/") {
 		return false
 	}
+	// FIXME: Remove it
+	// Becuase of some reason we don't have comment lines for Cloner.
+	// WTF?
+	glog.Errorf("-> %#v %#v", t, t.CommentLines)
```

Comment:
@lavalamp - because of some reason, CommentLines are empty for Cloner type:
https://github.com/kubernetes/kubernetes/pull/18156/files#diff-d45680f91eeeb7a8af4131da98a72f04R25

Any ideas what is happening here?